src/components/pages/Animations/FaceoffReveal.js

```diff
@@ -0,0 +1,294 @@
+import React, { useRef } from 'react';
+import { animated, useChain } from 'react-spring';
+import useCountdown from './AnimationComponents/useCountdown';
+import useLeftDrawbackCrash from './AnimationComponents/useLeftDrawbackCrash';
+import useRightDrawbackCrash from './AnimationComponents/useRightDrawbackCrash';
+import useEnlargeStoryResults from './AnimationComponents/useEnlargeStoryResults';
+import useEnlargeCenterToTopLeft from './AnimationComponents/useEnlargeCenterToTopLeft';
+import useEnlargeCenterToTopRight from './AnimationComponents/useEnlargeCenterToTopRight';
+import useEnlargeCenter from './AnimationComponents/useEnlargeCenter';
+import useUpFromBottom from './AnimationComponents/useUpFromBottom';
+import useEnlargeVs from './AnimationComponents/useEnlargeVs';
+import useNameWinner from './AnimationComponents/useNameWinner';
+import useDivAppear from './AnimationComponents/useDivAppear';
+import useCounter from './AnimationComponents/useCounter';
+import { useHistory } from 'react-router-dom';
+import useGoBackButton from './AnimationComponents/useGoBackButton';
+
+const FaceoffReveal = props => {
+  const setToggle = props.setToggle;
+  // we need to bring in faceoff data from FaceoffContent / FaceoffSubDisplay
+  console.log(
+    'animation reveal props:',
+    props.history.location.state.dynamicInfo
+  );
+  const dynamicInfo = props.history.location.state.dynamicInfo;
+
+  const history = useHistory();
+  // determine screen size to adjust image height
+  let screenWidth = window.screen.width;
+
+  // instantiating dynamic variables for display
+  let topAvatarHeight = 0;
+  let crashAvatarHeight = 0;
+  let crashImageSize = 0;
+  let winnerImageSize = 0;
+  let vsHeight = 0;
+  let matchupType = dynamicInfo.Type;
+  let userAvatar = dynamicInfo.Submission1.AvatarURL;
+  let opponentAvatar = dynamicInfo.Submission2.AvatarURL;
+  // winnerUserName will be determined by "dynamicInfo.Winner"
+  let winnerUserName = 'CAT-LADY';
+  let pointsAwarded = dynamicInfo.Points;
+  // dynamicBackgroundColor will be determined by which matchup is being animated--
+  // we may need to create a variable / key-value pair to track which faceoff is occurring
+  let backColorArray = ['#438eac', '#ffde3b', '#e97451', '#C9E952'];
+  let dynamicBackgroundColor = backColorArray[Math.floor(Math.random() * 4)];
+  //    hex codes:
+  //      4. boston-blue -- #438eac
+  //      3. bright-sun -- #ffde3b
+  //      2. burnt-sienna -- #e97451
+  //      1. conifer -- #C9E952
+  console.log('matchup type', matchupType);
```

Comment:
Since it's still a work in progress I think we can leave this console.log

Reply:
I agree. They should be cleaned up eventually, but these can be useful for the next cohort coming in. I know I used several leftover console logs from past teams to help me figure out where to even begin looking.